### PR TITLE
[Fix] New line between breaking changes and issues

### LINKED ISSUE #12: New line between breaking changes and issues
Hi @martinmcwhorter could do you see this behavior?

When u insert one breaking change and report one referenced issue commit is generate thus:

```
`BREAKING CHANGE: Example``fix #123`
```

Would be great if could be like that:

```
`BREAKING CHANGE: Example`
`fix #123`
```

### PR DESCRIPTION
**Pull request by xeptore**
Hey @martinmcwhorter!

This pull request is based on #13 with upgraded dependencies.

Fixes #12.

### CODE REVIEW

**Comment by xeptore** [comment]
One of the main causes for issues like this is the use of [`word-wrap`](https://www.npmjs.com/package/word-wrap) package, which causes some uncertainty when combined with new-lines in filter pipelines, as it seems it trims leading and following empty lines from its input.

A better idea, for future refactors, in my opinion, will be:
- Separating new-lines management (both insertions and deletions) into a separate module, which might run in the last phase to transform final commit message as whole (not in the middle of the process and for each answer)
- Adding a validator-like module to uniform the CommitLint configuration we receive from users to a logically correct configuration then mapping it into a new configuration object which is independent of CommitLint and passed to other parts of the application instead of the CommitLint `rules` configuration object
- Do not mutate user answers, e.g., commit body, after submission. Currently, if a user enters commit message body, and submits by pressing Enter key, they see a new leading empty line gets added to their input immediately. Having user input unchanged until the final processing step, which is kind of the same phase as in the first bullet point, will increase user experience.

File: .husky/prepare-commit-msg [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-exec < /dev/tty && git cz --hook || true
+exec </dev/tty && npx cz --hook || true
```
Comment (martinmcwhorter) [inline .husky/prepare-commit-msg]: Why is this changed from git to npx?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "commitiquette",
-  "version": "1.1.1",
+  "version": "1.2.0",
```
Comment (martinmcwhorter) [inline package.json:3]: We should just bump the patch as we are fixing a bug, not adding a new feature, so 1.1.2

**Comment by martinmcwhorter** [comment]
Overall it looks good to go. I added a question and note to the review. 

The coverage has decreased with this PR, please add tests for the additional functions to at least bring the coverage to the same percentage.

**Comment by xeptore** [comment]
@martinmcwhorter I added more tests for `footer-maker` module.

File: .husky/prepare-commit-msg (hunk 1 above)
Comment (xeptore) [reply .husky/prepare-commit-msg]: I had an error on my local machine regarding the `cz` not being a `git` command. I will revert if it is only my problem.

File: package.json (hunk 2 above)
Comment (xeptore) [reply package.json:3]: It is caused by a rebase from #13. It should be `1.2.1` IMHO.

Comment (martinmcwhorter) [reply package.json:3]: Ah sure. We can leave it 1.2.0 -- thats fine

File: .husky/prepare-commit-msg (hunk 1 above)
Comment (martinmcwhorter) [reply .husky/prepare-commit-msg]: Hmm.. not sure. Maybe cz is no longer a git plugin. Lemme look into it.

Comment (xeptore) [reply .husky/prepare-commit-msg]: I am receiving this error:
```txt
git: 'cz' is not a git command. See 'git --help'.
```

Comment (xeptore) [reply .husky/prepare-commit-msg]: Reverted back to using `git` (b4c8540a3e597599c880dd3d6247ec3d2078cf06).

**Comment by xeptore** [comment]
@martinmcwhorter also we need to merge #13 as it has dependency upgrades which caused the version bump to 1.2.0 which is itself was the cause of this merge request becoming 1.2.1.
